prow/monitoring/prometheus_operator/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+load("@io_bazel_rules_k8s//k8s:object.bzl", "k8s_object")
+
+k8s_object(
```

Comment:
You need to specify the cluster you want to schedule this into (cluster = gke_project_zone_cluster, or whatever the openshift, etc equivalent is)